### PR TITLE
Adding unit tests

### PR DESCRIPTION
**Pull request by rleguen-broadsign**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,26 +5,31 @@
...
     "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit",
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
     "core-js": "^3.6.5",
     "vue": "^3.0.0",
-    "vue-router": "^4.0.0-0",
-    "vuex": "^4.0.0-0"
+    "vue-router": "^4.0.0-beta.13",
```
Comment (TomasMorton) [inline package.json:16]: How forward of you

Comment (rleguen-broadsign) [reply package.json:16]: !!

It's just what the CLI did!

Comment (rleguen-broadsign) [reply package.json:16]: Even crazier:
```
"vue-jest": "^5.0.0-alpha.4"
```

Comment (kseniya-baluk-broadsign) [reply package.json:16]: Looks like only beta package is out to support `vue-3.3.0` at this moment
https://github.com/vuejs/vue-test-utils-next/releases

Same for `vue-jest`

https://openbase.io/js/vue-jest/versions

Vue3 has been released only Sept 18, 2020

We should keep track of it and update once stable packages are released.

File: tests/unit/example.spec.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { shallowMount } from "@vue/test-utils";
```
Comment (kseniya-baluk-broadsign) [inline tests/unit/example.spec.js:1]: I also tried to add **@testing-library/vue** but it has peer dependency to **vue-template-compiler**. And this compiler is only used for Vue 2 and seems to be deprecated for Vue 3. 

https://www.npmjs.com/package/vue-template-compiler
https://github.com/testing-library/vue-testing-library#installation

So using default **vue/test-utils**

**Comment by kseniya-baluk-broadsign** [comment]
Moving to Vue2

**Pull request closed without merging**